File: python/euler4.py
#!/usr/bin/env python3

result = -1
for x in range(999,100,-1):
	for y in range(999,100,-1):
		number = x * y
		if str(number) == str(number)[::-1] and number > result:
			result = number
			break

print(result)

File: php/euler4.php
#!/usr/bin/php
<?php

$result = -1;
for ($x = 100; $x < 1000; $x++) {
	for ($y = 100; $y < 1000; $y++) {
		$num = $x * $y;
		if (strval($num) == strrev(strval($num)) && $num > $result)
		{
			$result = $num;
			break;
		}
	}
}
echo $result . PHP_EOL;

?>